readthedocs/search/conf.py

```diff
@@ -0,0 +1 @@
+SEARCH_EXCLUDED_FILE = ['search.html', 'genindex.html', 'py-modindex.html']
```

Comment:
I think we can move this into the `search.documents` file, instead of needing another file for it.